ESMF/MAPL/Aerosol_GridComp/AeroCinderella_GridComp/Cinderella/FieldDictMap.F90

```diff
@@ -0,0 +1,10 @@
+module FieldDictMap_mod
```

Comment:
Please use the convention we have in the rest of MAPL.  "Dict" is Python.  gFTL is based on STL, where these are Maps.